### PR TITLE
chore: add npm semver calculator

### PR DESCRIPTION
**Pull request by Hongarc**
This page helps us find the version of that module on npm match range.

My English is not good so I want you to help me improve the message for this page.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -889,6 +889,7 @@
 - [RunKit](http://blog.tonicdev.com/2015/09/30/embedded-tonic.html) - Embed a Node.js environment on any website.
 - [RequireBin](http://requirebin.com) - Shareable JavaScript programs powered by npm and browserify.
 - [github-npm-stats](https://chrome.google.com/webstore/detail/github-npm-stats/oomfflokggoffaiagenekchfnpighcef) - Chrome extension that displays npm download stats on GitHub.
+- [npm semver calculator](https://semver.npmjs.com/) - Npm semantic version calculator
```
Comment (Phrynobatrachus) [inline readme.md]: "Semantic version calculator, useful for determining match range in `package.json`."

Comment (Hongarc) [reply readme.md]: I press delete my fork so I can't change pull request again.

I tried to create again in Hongarc@351fff5 but It does not work :(

Comment (Phrynobatrachus) [reply readme.md]: Close this and make a new PR, also make the description match exactly. You will need backticks around the package.json so it'll format in the Markdown output correctly. 

`[npm semver calculator](https://semver.npmjs.com/) - Semantic version calculator, useful for determining match range in `package.json`.`

Comment (Hongarc) [reply readme.md]: Because I see another `package.json` don't have backtick so I don't use, I will create a pull request again :)

**Pull request closed without merging**